lib/http/base.js

```diff
@@ -88,6 +88,44 @@ HttpClient.prototype.requestAsync = async function (options)
   return axios.request(requestOptions);
 };
 
+/**
+ * Issues an async HTTP request.
+ *
+ * @param {Object} options
+ *
+ * @returns {Object} an object representing the request that was issued.
+ */
+HttpClient.prototype.requestAsync = async function (options) {
```

Comment:
I propose to rebase the branch with current master. Method requestAsync had been added there before. It should be merged.

Reply:
Ok, will do